### ISSUE TITLE
Inline editor breaking when removing all the (multiline) text at once

### ISSUE THREAD

**Issue by amberv0** [tinymce/tinymce#2808]
#Steps to reproduce:

1. Go to https://www.tinymce.com/docs/demo/inline/
2. Click on main editable area there (starting with "This is an editable div element.")
3. Hit Backspace or Enter
4. See editing field collapsing to zero-height

The problem does not appear when there is only one line of text. But it happens once there are two or more lines. Also, surprisingly, after this collapse typing some text won't help, but hitting Backspace or Enter will.

**Comment by TrevorPT** [comment]
Sounds like it's related (or a duplicate) of [#2291]( Select-all then delete leaves a blank editor with an empty <p> and no <br> in Chrome #2291 )

**Comment by amberv0** [comment]
@TrevorPT yes, seems to be related. Might even be a duplicate, but for some reason behavior is somewhat different in our two examples. When typing in your fiddle after removing all the text we do see the text entered, but not in my example. The difference is caused by the style "border: 1px solid black;" that you added, and this is surpising
